style.css

```diff
@@ -0,0 +1,99 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  text-align: center;
+  background: rgb(241, 69, 98);
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+#container > * {
+  background: rgb(27, 228, 168);
+  padding: 20px;
+  border-radius: 5px;
+}
+#container > *:hover {
+  background: rgb(243, 162, 243);
+  transition: 0.3s;
+}
+#container > *:hover img {
+  transform: translateY(-5px);
+  transition: 0.3s;
+}
+
+#container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-rows: minmax(200px, auto);
+  grid-gap: 8px;
+  grid-template-areas:
+    "one one two three"
+    "four five two six"
+    "four sev sev sev";
+}
+.one {
+  grid-area: one;
+}
+.one h2 {
+  padding-bottom: 10px;
+}
+.two {
+  grid-area: two;
+}
+.three {
+  grid-area: three;
+}
+.four {
+  grid-area: four;
+}
+.five {
+  grid-area: five;
+}
+.six {
+  grid-area: six;
+}
+.seven {
+  grid-area: sev;
+}
+.seven h2 {
+  font-size: 50px;
+  margin-top: 40px;
+}
+.seven p {
+  font-size: 20px;
+}
+@media screen and (max-width: 768px) {
+  #container {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-auto-rows: minmax(200px, auto);
```

Comment:
Can you explain how values `minimax(200px, auto)` effect

Reply:
if the content inside the grid is less than 200px so it takes the min-height of grid as 200px in this case there is no role of auto height but if content inside the grid is more than 200px so takes the content height. @Abbas4tech